### DISCUSSION TITLE
Use 'All schemas' in database connection for DDEV

### DISCUSSION THREAD

**Discussion by IT-Cru** [php-perfect/ddev-intellij-plugin#121] [Ideas]
Currently the DDEV plugin seems to use always only the default database schema for its DDEV DB connection. This is working fine, if you only use default DB of your DDEV env.

But in more complex scenarios, where you use multiple databases in your DDEV project, you always have to switch schemas configuration in DDEV database properties to use 'All schemas' instead of 'Default schema' to see all your databases you have in your DDEV project. This currently needs to be done on every DDEV project start/restart.

**Comment by nico-loeber** [comment]
Thanks for pointing this out, @IT-Cru!
This is not the intedned behavior, I will take a look at this and will try if I can enable the "All schemas" setting by default.

**Comment by nico-loeber** [comment]
This was added in 1.0.4 so I am closing this discussion. :)
